plugins/config/repository.yaml

```diff
@@ -24,6 +24,103 @@ repositories:
 # wildcards are allowed in name, entries are processed in top-down order
 # first entry matching the name is used
 projects:
+    # Use nightlies in all main branches
+    - name: ignition-fortress
+      repositories:
+          - name: osrf
+            type: stable
+          - name: osrf
+            type: nightly
+    - name: ignition-cmake3
```

Comment:
are we bumping ign-cmake on Fortress or this have a different propose?

Reply:
No, we're not bumping it (yet at least). My idea was to use nightlies for all the `main` branches for consistency. It's easier to always keep all main branches using nightlies than having to come back here to enable them just when they're needed.

But I can remove it if we think it's not a good idea.

Reply:
Removed the non-fortress versions until I can articulate better my arguments for standardizing the main branches: 303982b